packages/components/src/components/hds/advanced-table/index.hbs

```diff
@@ -42,50 +45,50 @@
             @tooltip={{column.tooltip}}
             @isVisuallyHidden={{column.isVisuallyHidden}}
             @isExpandable={{column.isExpandable}}
-            @onClickToggle={{this.onExpandAllClick}}
-            @isExpanded={{this._expandAllButtonState}}
-            @hasExpandAllButton={{this.hasNestedRows}}
-            @didInsertExpandButton={{this.didInsertExpandAllButton}}
-            @willDestroyExpandButton={{this.willDestroyExpandAllButton}}
-          >{{column.label}}</Hds::AdvancedTable::Th>
+            @onClickToggle={{this._tableModel.toggleAll}}
```

Comment:
Suggestion: the type for this arg optionally accepts a newValue, since thats not needed anymore we could remove it.